File: CS/TestMyTreeListForum/MyTreeListPainter.cs
using System;
using System.Collections.Generic;
using DevExpress.XtraTreeList.Painter;



namespace DevExpress.MyControl
{
    public class MyTreeListPainter : TreeListPainter
    {
        protected internal MyTreeListPainter() 
            :base()
        {	
		}



        public override ITreeListPaintHelper CreatePaintHelper()
        {
            return new MyTreeListPaintHelper();
        }
    }
}

File: VB/TestMyTreeListForum/MyTreeListPainter.vb
Imports System
Imports System.Collections.Generic
Imports DevExpress.XtraTreeList.Painter



Namespace DevExpress.MyControl
	Public Class MyTreeListPainter
		Inherits TreeListPainter

		Protected Friend Sub New()
			MyBase.New()
		End Sub



		Public Overrides Function CreatePaintHelper() As ITreeListPaintHelper
			Return New MyTreeListPaintHelper()
		End Function
	End Class
End Namespace
